phpunit/blocks/block-navigation-block-hooks-test.php

```diff
@@ -92,4 +92,22 @@ public function test_block_core_navigation_update_ignore_hooked_blocks_meta_pres
 			'Block was not added to ignored hooked blocks metadata.'
 		);
 	}
+
+	/**
+	 * @covers ::gutenberg_block_core_navigation_update_ignore_hooked_blocks_meta being called via the REST API.
```

Comment:
Just

```suggestion
	 * @covers ::gutenberg_block_core_navigation_update_ignore_hooked_blocks_meta
```

by convention, I think